canonical-websites/www.ubuntu.com
# Updated /support with ESM and general clean-ups

## Done

- Updated /support with ESM and general clean-ups
- added the ITstrategem logo
- removed include that wasn't used elsewhere

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/support](http://0.0.0.0:8001/support)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- compare to the [copy doc](https://docs.google.com/document/d/1liY2ulrI3JiICSJquTn9cH0Hu-R6H3ufmEmYhive-QI/edit#heading=h.14i65plmd9q2)


## Issue / Card

Fixes #4616
